### PR TITLE
Refactor converting ODK XML to OSM XML

### PR DESCRIPTION
**Pull request by rsavoye**
The code for converting the raw ODK XML instance files to OSM has not kept up with other improvements in the conversion process. This brings odk2osm.py in sync with the other conversion utilities, while also adding support for select_multiple.

### CODE REVIEW

**Comment by spwoodcock** [comment]
Conflicts must be resolved before merge 👍

**Comment by rsavoye** [comment]
Major refactoring. The format parsers for CSV, JSON, or XML are now in a single class, so next several files can be deleted. This reduces code duplication, and improves maintenance. Now the odk2osm.py script takes any of the 3 support formats instead of separate scripts.

File: osm_fieldwork/ODKInstance.py [hunk 1]
```diff
@@ -50,6 +49,7 @@ def __init__(
         """
         self.data = data
         self.filespec = filespec
+        self.ignore = ["today", "start", "deviceid", "nodel", "instanceID"]
```
Comment (spwoodcock) [inline osm_fieldwork/ODKInstance.py:52]: Probably nodel --> model?

**Review by spwoodcock — APPROVED** [review]
Nice! json2osm is back 😁

File: osm_fieldwork/ODKInstance.py (hunk 1 above)
Comment (rsavoye) [reply osm_fieldwork/ODKInstance.py:52]: ODKInstance.py is about to be deleted, as now parsers.py is used instead. Also now the tags to ignore are in the xforms.yaml file.

Comment (rsavoye) [reply osm_fieldwork/ODKInstance.py:52]: well, json2osm.py is about to be deleted, now odk2osm.py handl;es all 3 formats from ODK.

**Pull request merged**